WtWSMS/common/script_values/BP_war_values.txt

```diff
@@ -1,8 +1,30 @@
-﻿### Constants
-migration_war_fame_base = major_prestige_value
+﻿#### Reconquest War Stuff
+imperial_reconquest_war_fame_base_cost = 100
+imperial_reconquest_war_gold_base_cost = 500
+successor_reconquest_war_fame_base_cost = 500
+successor_reconquest_war_gold_base_cost = 500
```

Comment:
Is it possible to make this scale like it did in CK2 rather than being a constat 500? With income increasing or large realms 500 like an empire can become a smaller sum.